lib/prism/lex_compat.rb

```diff
@@ -240,10 +242,10 @@ class IgnoredNewlineToken < Token
       def ==(other) # :nodoc:
         return false unless self[0...-1] == other[0...-1]
 
-        if self[4] == Ripper::EXPR_ARG | Ripper::EXPR_LABELED
-          other[4] & Ripper::EXPR_ARG | Ripper::EXPR_LABELED > 0
+        if self[3] == Ripper::EXPR_ARG | Ripper::EXPR_LABELED
+          other[3] & Ripper::EXPR_ARG | Ripper::EXPR_LABELED > 0
         else
-          self[4] == other[4]
+          self[3] == other[3]
```

Comment:
I couldn't quite figure out a good way to trigger these lines, but I believe these should be 3 not 4.

Reply:
Agreed.